.travis.yml

```diff
@@ -10,6 +10,8 @@ os:
 sudo: required
 
 env:
+  - IMAGEMAGICK_VERSION=6.7.7-10
+  - IMAGEMAGICK_VERSION=6.7.7-10 CONFIGURE_OPTIONS=--enable-hdri
```

Comment:
Can we add this to `.appveyor.yml`, too?

Reply:
make sense. it sounds good👍

Reply:
After https://github.com/rmagick/rmagick/pull/340, we will add appveyor configuration.